src/cards/serializers.py

```diff
@@ -31,6 +31,10 @@ class Meta:
 class CardSerializer(serializers.ModelSerializer):
     card_stats = CardStatSerializer()
     slug = serializers.CharField(required=False)
+    help_links = serializers.ListField(
```

Comment:
The doc specify that that you should not invoke it directly but use `serializers.CharField(many=True)` instead.
Any reason you didn't?

Reply:
Hmmm I'd like to know about the doc saying that?
I just found that: http://www.django-rest-framework.org/api-guide/fields/#listfield
But I didn't find what you are talking about?

Reply:
Here: http://www.django-rest-framework.org/api-guide/serializers/#listserializer